qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_module.py

```diff
@@ -243,6 +243,24 @@ def get_measurement_summary(status_array: np.ndarray) -> str:
 
     return summary
 
+
+def convert_dummy_val_to_nan(param: _FMTResponse):
+    """
+    Converts dummy value to NaN. Instrument may output dummy value (
+    199.999E+99) if measurement data is over the measurement range. Or the
+    sweep measurement was aborted by the automatic stop function or power
+    compliance. Or if any abort condition is detected. Dummy data
+    199.999E+99 will be returned for the data after abort."
+
+    Args:
+        param: This must be of type named tuple _FMTResponse.
+
+    """
+    for index, value in enumerate(param.value):
+        if value == 199.999e99:
```

Comment:
i believe that JohnW used a more generous condition for nan substitution in measurements - ` > 1e99`. shall we do that? using `>` would also let us skip possible floating-point precision issues.